src/main/scala/firrtl/Compiler.scala

```diff
@@ -449,7 +449,7 @@ object CompilerUtils extends LazyLogging {
         case ChirrtlForm =>
           Seq(new ChirrtlToHighFirrtl) ++ getLoweringTransforms(HighForm, outputForm)
         case HighForm =>
-          Seq(new IRToWorkingIR, new ResolveAndCheck, new transforms.DedupModules, new HighFirrtlToMiddleFirrtl) ++
+          Seq(new IRToWorkingIR, new ResolveAndCheck, new firrtl.transforms.DedupModules, new HighFirrtlToMiddleFirrtl) ++
```

Comment:
why this happens? `Compiler` is in firrtl package, and `transform` is in the namespace, I think this line shouldn't be changed.

Reply:
It doesn't compile if I do not change it. But yeah, honestly I do not know why...

Reply:
Maybe `transforms` somehow gets overwritten